step-capstone/src/Components/Hotel.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import {Grid, Typography} from '@material-ui/core'
+
+export default function Hotel(data) {
+
+  /**
+   * Takes data in the form: 
+   * type : 'hotel
+   * name : ' Hotel ZED ' 
+   * location : 'London'
+   * checkIn : December 17, 2020 15:30:00
```

Comment:
Consider including timezone here (if you're planning on including it in the spec go ahead and put it in the comment as well).